src/css/generic/_reset.scss

```diff
@@ -1,22 +1,20 @@
 html,
 body {
-  height: 100%;
+  min-height: 100%;
   margin: 0;
 }
 
 html {
   text-size-adjust: none;
-}
 
-@media (prefers-reduced-motion: no-preference) {
-  html {
+  @media (prefers-reduced-motion: no-preference) {
     scroll-behavior: smooth;
   }
 }
 
 body {
-  -webkit-font-smoothing: antialiased;
```

Comment:
Global font smoothing is not recommended. This comes from MDN which references these two articles:
- [Please Stop “Fixing” Font Smoothing](https://usabilitypost.com/2012/11/05/stop-fixing-font-smoothing/) by Dmitry Fadeyev
- [Laissez-faire Font Smoothing and Anti-aliasing](https://www.zachleat.com/web/font-smooth/) by Zach Leatherman

Reply:
Excuse me while I update all my projects, thanks for the links!